std/algorithm/comparison.d

```diff
@@ -925,7 +925,7 @@ range of range (of range...) comparisons.
 
 @safe pure unittest
 {
-    import std.utf : byChar, byWchar, byDchar;
+    import std.utf : byChar, byDchar, byWchar;
```

Comment:
I'm not too worried about the few hand-optimizations that could be done. I'd say sorted is the simple and effective way.